build.gradle

```diff
@@ -58,9 +58,9 @@ subprojects { subproject ->
 	apply plugin: 'maven-publish'
 	apply plugin: 'signing'
 
-	java.sourceCompatibility = JavaVersion.VERSION_1_8
-	java.targetCompatibility = JavaVersion.VERSION_1_8
-	
+	java.toolchain.languageVersion.set(
```

Comment:
I'm not 100% sure this is what we want.  I think for the foreseeable future, when we cut WALA releases, the target compatibility will be 1.8.  So, even when _running_ on a Java 11+ VM, we may still want the source and target compatibility flags for `javac` set to 1.8, to mimic running an actual WALA release on the more modern VM.  What do you think?

Reply:
> even when _running_ on a Java 11+ VM, we may still want the source and target compatibility flags for `javac` set to 1.8, to mimic running an actual WALA release on the more modern VM. What do you think?

Bleh. But OK, I see what you mean. I've updated the code in this pull request to revert to the older style of setting `java.sourceCompatibility` and `java.targetCompatibility` to `JavaVersion.VERSION_1_8` in all subprojects except for `com.ibm.wala.java11`. In that last subproject, we need to set `sourceCompatibility` to `VERSION_11`, since we genuinely do have source code here that uses post-Java-1.8 syntax. Once we've done that, Eclipse import fails unless we also set `targetCompatibility` to `VERSION_11` in this subproject as well.

Now I'm having second thoughts, though. If your concern is what happens "when we cut WALA releases", then wouldn't it be sufficient to always cut WALA releases using a Java 1.8 toolchain? That would have been the default in my `java.toolchain`-using code: the code you reviewed when making this comment. If we moved that default to some later version in the future, we could have used `./gradlew -PjavaToolchainLanguageVersion=1.8` to explicitly request a Java 1.8 toolchain when building releases. If that succeeded at all, then presumably it would produce Java-1.8-compatible artifacts. Shouldn't that be everything we need, even when using the new Java toolchain API?